pants_plugins/sendwave/pants_docker/package.py

```diff
@@ -57,36 +57,33 @@
 logger = logging.getLogger(__name__)
 
 
-def _build_tags(
-    target_name: str, tags: List[str], registry: Optional[str]
-) -> List[str]:
+def _build_full_target_name(target_name: str, registry: Optional[str]) -> str:
+    """Generate a Docker target name that specifies an optional registry, if one exists."""
+    return f"{registry}/{target_name}" if registry else target_name
```

Comment:
I factored out this function because we want to reuse the full target name when we're adding the remote cache arguments to the build command.